File: Silver/Cooper/Android/ActionBar/SearchActivity.swift
import java.util
import android.app
import android.content
import android.os
import android.util
import android.view
import android.widget

public class SearchActivity : Activity {
	
	public override func onCreate(savedInstanceState: Bundle!) {
		
		super.onCreate(savedInstanceState);
		
		let textView = TextView(self);
		textView.TextSize = 40;
		textView.Text = "Search Activity";
		ContentView = textView;
		
		getActionBar().setDisplayHomeAsUpEnabled(true);
	}
}

File: Hydrogene/Cooper/Android/ActionBar/com.remobjects.actionbar/SearchActivity.cs
using java.util;
using android.app;
using android.content;
using android.os;
using android.util;
using android.view;
using android.widget;

namespace com.remobjects.actionbar
{
	public class SearchActivity : Activity
	{
		public override void onCreate(Bundle savedInstanceState)
		{
			base.onCreate(savedInstanceState);

			TextView textView = new TextView(this);
			textView.TextSize = 40;
			textView.Text = "Search Activity";
			ContentView = textView;
			getActionBar().setDisplayHomeAsUpEnabled(true);
		}
	}
}